css/styles.css

```diff
@@ -90,13 +90,17 @@ body {
     background-color: rgba(255, 255, 255, .75);
 }
 
+.header-fixed {
+    background-color: pink;
```

Comment:
Would we want to use Sass variables for colors?

Reply:
no, I was testing this out, I'll remove this asap